packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -30,13 +30,8 @@ import 'localizations.dart';
 const double _kBackGestureWidth = 20.0;
 const double _kMinFlingVelocity = 1.0; // Screen widths per second.
 
-// An eyeballed value for the maximum time it takes for a page to animate forward
-// if the user releases a page mid swipe.
-const int _kMaxDroppedSwipePageForwardAnimationTime = 800; // Milliseconds.
-
-// The maximum time for a page to get reset to it's original position if the
-// user releases a page mid swipe.
-const int _kMaxPageBackAnimationTime = 300; // Milliseconds.
+// The duration for a page to animate when the user releases it mid-swipe.
+const Duration _kDroppedSwipePageAnimationDuration = Duration(milliseconds: 350);
```

Comment:
Do we always want to use the same duration for forward and back now? The videos you have are only for if the drag is not far enough to start the pop.

Reply:
I am certain that regardless of moving forward or backward, and regardless of the drag distance, the animation duration on iOS native is always the same. I have supplemented the video of moving forward.